python/foglamp/services/core/server.py

```diff
@@ -419,14 +419,14 @@ def _start_core(cls, loop=None):
             # registering now only when service_port is ready to listen the request
             # TODO: if ssl then register with protocol https
             cls._register_core(host, cls.core_management_port, service_server_port)
-            print("(Press CTRL+C to quit)")
-            try:
-                loop.run_forever()
-            except KeyboardInterrupt:
-                pass
-            finally:
-                cls.stop(loop)
-            loop.close()
+            # print("(Press CTRL+C to quit)")
+            # try:
+            loop.run_forever()
+            # except KeyboardInterrupt:
+            #     pass
+            # finally:
+            #     cls.stop(loop)
+            # loop.close()
         except (OSError, RuntimeError, TimeoutError) as e:
```

Comment:
@MarkRiddoch Please let me know, we want to remove this ctrl+c ? as we want to start with scripts only.

Reply:
Yes, we can remove that code